test/functional/interface_rpc.py

```diff
@@ -67,10 +80,23 @@ def test_http_status_codes(self):
         expect_http_status(404, -32601, self.nodes[0].invalidmethod)
         expect_http_status(500, -8, self.nodes[0].getblockhash, 42)
 
+    def test_work_queue_exceeded(self):
+        self.log.info("Test work queue exceeded")
```

Comment:
nit: Mind rewording to make it consistent with other log messages
```suggestion
        self.log.info("Testing work queue exceeded...")
```

Reply:
Force-pushed to implement review suggestions ([diff](https://github.com/bitcoin/bitcoin/compare/db0d4605b9e57230a02f8b620f93e5e4d191972c..8dd5946c0b7aa8f3976b14a5de4ce84b80a9c32a))